src/requests/cache/sorted_set/sorted_set_put_elements.rs

```diff
@@ -41,15 +128,15 @@ use crate::{CacheClient, CollectionTtl, IntoBytes, MomentoResult};
 /// # Ok(())
 /// # })
 /// # }
-pub struct SortedSetPutElementsRequest<S: IntoBytes, E: IntoBytes> {
+pub struct SortedSetPutElementsRequest<S: IntoBytes, E: IntoSortedSetElements> {
```

Comment:
you should also update the signature in `cache_client.sorted_set_put_elements`, that's actually where we get the most value from this

Reply:
Thanks for the catch. Fixed in [ee60e18](https://github.com/momentohq/client-sdk-rust/pull/207/commits/ee60e18a7bf3b7b0fa2ba2a8bd4cf510a9dcda16)